From 27dcc3a583e5488ee70dcd0299646cc33faf8a21 Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Sun, 26 Oct 2014 11:48:15 +0000
Subject: [PATCH] kernel-modules: Drop explicit modules that are found with
 block_module_filter

The usb_storage, nvme and sdhci_acpi modules are discovered with the
block_module_filter so there's no need to explicitly list them here.

Signed-off-by: <pbrobinson@gmail.com>
---
 modules.d/90kernel-modules/module-setup.sh | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/modules.d/90kernel-modules/module-setup.sh b/modules.d/90kernel-modules/module-setup.sh
index 83b8995..2fa263c 100755
--- a/modules.d/90kernel-modules/module-setup.sh
+++ b/modules.d/90kernel-modules/module-setup.sh
@@ -44,8 +44,7 @@ installkernel() {
         instmods yenta_socket scsi_dh_rdac scsi_dh_emc \
             atkbd i8042 usbhid hid-apple hid-sunplus hid-cherry hid-logitech \
             hid-logitech-dj hid-microsoft hid-lcpower firewire-ohci \
-            pcmcia usb_storage nvme hid-hyperv hv-vmbus \
-            sdhci_acpi hyperv-keyboard
+            pcmcia hid-hyperv hv-vmbus hyperv-keyboard
 
         if [[ "$(uname -p)" == arm* ]]; then
             # arm specific modules
